src/hooks/private/index.ts

```diff
@@ -0,0 +1,11 @@
+export * from './useBoolean';
+export * from './useCheckbox';
+export * from './useCloseOnTimeout';
+export * from './useConditionallyControlledState';
```

Comment:
Why is this private hook? It's general, it can be used by any app.

Reply:
it was not exported outside

Reply:
Ok ✅ 
Seems like it will be good to make it public.